server/sendq.go

```diff
@@ -74,16 +76,18 @@ func (sq *sendq) internalLoop() {
 				} else {
 					c.pa.reply = nil
 				}
-				var msg []byte
+				msg = msg[:0]
```

Comment:
Since we know how much we will need here, if len(pm.hdr) + len(pm.msg) > cap(_msg) we should allocate and avoid second copy?

Reply:
The variable msg is new each signal, so it would only last for a single pop, and we would still have to unnecessarily copy twice. Also think of a case where each msg is slightly bigger.. Then every message has 2 copies to expand.

Reply:
ok so I see you changed its scope. Good deal.